_data/simple-icons.json

```
@@ -6387,6 +6387,12 @@
             "hex": "101113",
             "source": "https://github.com/palantir/conjure/blob/master/docs/media/palantir-logo.svg"
         },
+        {
+            "title": "Palo Alto Networks",
+            "hex": "F04E23",
+            "source": "https://www.paloaltonetworks.com/company/brand",
+            "guidelines": "https://www.paloaltonetworks.com/content/dam/pan/en_US/partners/PAN-brand-guidelinesv22-lores.pdf"
```

I'd recommend changing this to the brand page, as opposed to the specific PDF. We don't want to not have guidelines available just because they change the URL.

```suggestion
            "guidelines": "https://www.paloaltonetworks.com/company/brand"
```

---

Okay, I am changing it to brand page